src/Export/VTK.jl

```diff
@@ -1,3 +1,75 @@
+struct VTKStream{VTK<:WriteVTK.DatasetFile, DH<:Union{DofHandler,Grid}}
+    vtk::VTK
+    grid_or_dh::DH
+end
+"""
+    Base.close(vtks::VTKStream)
+
+Close the vtk stream and save the data to disk. 
+"""
+Base.close(vtks::VTKStream) = WriteVTK.vtk_save(vtks.vtk)
```

Comment:
`close_vtk`?

Reply:
I chose the function names to be independent of vtk to make the interface suitable for potential future, or user, extensions. One example could be a JLD2-interface (cc: @koehlerson). Could use `close_stream` if you prefer not overloading `close`?

Reply:
Was just thinking it should match `open_vtk`.

Reply:
Fair point! 
Perhaps `open_vtk` should be replaced by `VTKStream` instead?

Reply:
But is it "ok" to have a constructor in a do-block?

Reply:
My thinking was that the user (since direct JLD2 support in Ferrite doesn't seem to be on the roadmap currently) can define e.g., 
```julia
struct JLDStream{T<:"JLD2File?"}
   jldfile::T
end
``` 
and then define e.g.
* `write_solution(jld::JLDStream, dh, u)`
* `write_nodedata(jld::JLDStream, grid, data, name)`
* `write_celldata(jld::JLDStream, grid, data, name)`
* ... 

Where less complicated transforms than for vtk are required for e.g. tensorial data since `Vector{Tensor}` can be saved directly.

This makes it possible to use the same code for export independent of the backend.

If we would make a JLD2 extension (or another saving backend) in the future, this could be included there.